source/beginners_guide/raw_beginners_guide.rst

```diff
@@ -184,7 +184,62 @@ Supported Modsets
+
+         *  - `ACE <https://steamcommunity.com/workshop/filedetails/?id=463939057>`_
+            -  | ACE is a mod that adds a log of in depths modular systems to Arma like for example a medical system, ballistics and more.
+               | Keep in mind that some mods like RHS need their respective `ACE Compatibility Mods <https://steamcommunity.com/id/acemod/myworkshopfiles/?appid=107410>`_ loaded so they work properly. Adjust your modlist accordingly.
+
+         *  - `TFAR (outdated) <https://steamcommunity.com/sharedfiles/filedetails/?id=620019431>`_
+            -  | TFAR is a mod which implements a more realistic radio communication by combining Arma 3 and TeamSpeak. It implements SR and LR radios for infantry and vehicles.
+               | This version is outdated and superseded by TFAR BETA.
+
+         *  - `TFAR BETA <https://steamcommunity.com/sharedfiles/filedetails/?id=894678801>`_
+            - TFAR BETA is the successor of TFAR, adapting the functionality to newer BI functions, fixing bugs and adding new settings and options.
+
+         *  - `ACRE2 <https://steamcommunity.com/sharedfiles/filedetails/?id=751965892>`_
+            - ACRE2, like TFAR / TFAR BETA is a mod which combines Arma 3 with TeamSpeak. It is highly customizable, has additional features like taking the terrain of a map into account for the quality of the radio transmission and more. It is more in depths, requires more time to learn, but adds a nice level of realism. For CDLC maps it requires `ACRE2 Compatibility Mods <https://steamcommunity.com/profiles/76561198323575101/myworkshopfiles/?appid=107410>`_.
+
+         *  - `KAT - Advanced Medical REWRITE <https://steamcommunity.com/workshop/filedetails/?id=2020940806>`_
+            - KAT - Advanced Medical REWRITE is a mod which adds a very in depths medical system, covering different not only the breathing apperatus and the cardiovascular system but also adds surgery and more to the game.
+
+   .. dropdown:: Additional Vehicle Mods
+      :class-title: header-3-light
+      :class-container: sd-card-3
+
+      Antistasi has compatibility with the following mods which add to the available civilian vehicle pool if loaded properly and enabled in the Startup GUI.
+
+      - `D3S Cars pack <https://steamcommunity.com/workshop/filedetails/?id=1639607571>`_
+      - `Ivory Car Pack [Official] <https://steamcommunity.com/sharedfiles/filedetails/?id=1888644057>`_
+      - `RDS Civilian Pack <https://steamcommunity.com/workshop/filedetails/?id=612930542>`_
+      - `TCGM_BikeBackpack <https://steamcommunity.com/sharedfiles/filedetails/?id=2096950604>`_
+
+   .. dropdown:: Forbidden Mods
+      :class-title: header-3-light
+      :class-container: sd-card-3
+
+      There are quite some mods which are NOT compatible with Antistasi.
```

Comment:
Make NOT a different color

Reply:
Not necessary. It would be the only different-coloured word and it being capitalized and within a section called "Forbidden Mods" should be significant enough.

Reply:
has been highlighted as complete paragraph